org.ultimatepp.TheIDE.yml

```diff
@@ -0,0 +1,115 @@
+app-id: org.ultimatepp.TheIDE
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+sdk-extensions: ["org.freedesktop.Sdk.Extension.llvm17"]
+command: run-ide.sh
+finish-args:
+  - --socket=x11
+  - --share=network
+  - --share=ipc
+  - --filesystem=home
+  - --filesystem=host
```

Comment:
these are very broad and redundant.

Reply:
I didn't now that host also covers home. However, after removing home it looks like the access to that directory works as expected. home removed in latest version.

--filesyste=host premision is needed since the application is IDE. Here is the quotation from PR description "access to host directory is needed. The compilation done by IDE is executed on host not within sandbox. So, broad access is needed.". I also notcied that other IDE's such as VS Code also using this permission.

Reference:
- https://github.com/flathub/com.visualstudio.code/blob/c2704e2c9c31ba12f8266fc27d5c73f6e68a51f4/com.visualstudio.code.yaml#L20

Reply:
> I also notcied that other IDE's such as VS Code also using this permission.

two wrongs don't make a right.

as for "host" I don't see why you'd need anything out of the home directory.

Reply:
Without access to host filesystem. This Flatpak doesn't make sense. TheIDE It is whole compilation and language environment. You are compiling your program on host not within sandbox.

Another example (Code::Blocks) for C++. It uses both --filesystem=home and --filesystem=host. I don't understand why I can not use it here and they can. Do you have a policy about it?

References:
- https://github.com/flathub/org.codeblocks.codeblocks/blob/d660f090f2e9726d1476d73ac200c7b74f4b8867/org.codeblocks.codeblocks.yaml#L15